Chore/ update readme.md (#75)
Solvro/web-juwenalia: README.md

- [@czaja307](https://github.com/czaja307) - Devops

## Technologies used

- Next.js
- React.js
- TypeScript
- Tailwind CSS
- Coolify

## Links

[![docs.solvro.pl](https://i.imgur.com/fuV0gra.png)](https://docs.solvro.pl)

## Development

### 1. Clone the Repository

```bash
git clone https://github.com/Solvro/web-juwenalia.git
```

### 2. Install Dependencies

```bash
cd web-juwenalia
npm install
```

### 3. Configure Environment

Create an `.env` file with the following content:

```env
FACEBOOK_ACCESS_TOKEN=
FACEBOOK_APP_ID=
FACEBOOK_APP_SECRET=
```

### 4. Run the Project

```bash
npm run dev
```

### 5. View the Application

Open your browser and navigate to [http://localhost:3000](http://localhost:3000).

## Working with git

### Github Solvro Handbook

<https://docs.solvro.pl/guides/github>

### SSH

If you are a Windows user, follow this [tutorial](https://www.youtube.com/watch?v=vExsOTgIOGw) to connect via SSH

### Building a new feature

1. Checkout and update main branch

```bash
   git checkout main
   git pull origin main
   git fetch
```

2. Create new feature branch

```bash
   git checkout -b feat/x_my_feature_branch
```

> > 'x' stands for issue number; this command will create and checkout a new branch named feat/x_my_feature_branch 3. Commit your changes:

Reviewer: zgubił się newline

```suggestion
> 'x' stands for issue number; this command will create and checkout a new branch named feat/x_my_feature_branch

3. Commit your changes:
```

> # Juwenanlia Wrocław Razem

Reviewer: ```suggestion
# Juwenalia Wrocław Razem
```
literówka

> **The official website**

Reviewer: ```suggestion
## Official website
```
Gramatyka + [formatowanie](https://github.com/DavidAnson/markdownlint/blob/main/doc/md036.md)

Author: Tutaj niekoniecznie, moim zdaniem nie ma potrzeby robienia z tego nagłówka bo to jest tylko dopisek pod nagłówkiem. 
+ obie formy są poprawne z tego co się orientuję ale bez the może brzmi trochę bardziej organicznie

Reviewer: Co do nagłówka kierowałem się markdownlintem, on mocno odradza używanie samego bolda w całej linijce, i woli używać headerow, ale nie wiem jak semantycznie inaczej można takiego "subtitle" zrobić, więc obojętne raczej

Jak dla mnie to "the" wydaje się bardzo nienaturalne i zdecydowanie lepiej bez. Z "the" to brzmi jak początek zdania, nie uzywa się tego tak w tym kontekście

Author: Spoko pozbywam się the, z boldem się wstrzymuje

![Homepage](https://i.imgur.com/69i20kk.jpeg)

## 🇵🇱

> #### Witamy w repozytorium oficjalnej strony internetowej Juwenaliów #wrocławrazem!

Author: + muszę przejrzeć tą strukturę po tych modyfikacjach
+ #wroclawrazem oni stostuha jako część nazwy a nie faktycznie tag

> #### Witamy w repozytorium oficjalnej strony internetowej Juwenaliów #wrocławrazem!

Reviewer: ok, linter mi krzyczał że nagłówki nie powinny się kończyć interpunkcją i uznałem że może to być proste obejście

> #### Witamy w repozytorium oficjalnej strony internetowej Juwenaliów #wrocławrazem!

Author: Czaje

> Aplikacja jest owocem współpracy członków Koła Naukowego Solvro oraz Samorządu Studenckiego Politechniki Wrocławskiej. Stanowi oficjalny kanał komunikacji pomiędzy organizatorami wydarzenia oraz jego uczestnikami. Jej celem jest szybkie i wygodne dostarczenie wszelkich niezbędnych informacji dla imprezowiczów - bez potrzeby przeszukiwania social mediów. Harmonogram, mapa, artyści - wszystko w jednym miejscu! Co więcej, dzięki integracji z profilem na Facebooku organizatora, aplikacja zapewnia, że wszyscy uczestnicy zawsze będą na bieżąco ze wszystkim, co dzieje się wokół Juwenaliów.

Reviewer: ```suggestion
Aplikacja jest owocem współpracy członków Koła Naukowego Solvro oraz Samorządu Studenckiego Politechniki Wrocławskiej. Stanowi oficjalny kanał komunikacji pomiędzy organizatorami wydarzenia oraz jego uczestnikami. Jej celem jest szybkie i wygodne dostarczenie wszelkich niezbędnych informacji dla imprezowiczów—bez potrzeby przeszukiwania social mediów. Harmonogram, mapa, artyści—wszystko w jednym miejscu! Co więcej, dzięki integracji z profilem na Facebooku organizatora, aplikacja zapewnia, że wszyscy uczestnicy zawsze będą na bieżąco ze wszystkim, co dzieje się wokół Juwenaliów.
```
stylistyka

Author: A ja wolę taką stylistykę🥴

> ## 🇬🇧

Reviewer: ```suggestion
### 🇬🇧
```
[formatowanie](https://github.com/DavidAnson/markdownlint/blob/v0.36.1/doc/md001.md)

Author: Tu trochę nie czaję, to jest lvl drugi chyba tak jak powinno być?

Author: Dobra już rozumiem xd To generalnie dopisek the official website celowo nie jest nagłówkiem więc struktura nagłówków pozostaje

#### Welcome to the repository of the Juwenalia #wrocławrazem official website!

> The app is the result of collaboration between the members of Solvro Science Club and Student's Union of Wrocław University of Science and Technology. It serves as the official communication channel between the event organizers and its participants. Its main goal is to quickly and conveniently provide all the essential information for festival goers - without the need to search through social media. Event schedule, map, artists - all that in one place! What's more, thanks to integration with the organizer's Facebook account, we ensure that You will always stay up-to-date!

Reviewer: ```suggestion
This app is the result of collaboration between members of the Solvro Science Club and the Students' Union of Wrocław University of Science and Technology. It serves as the official communication channel between the event organizers and its participants. Its main goal is to quickly and conveniently provide all the essential information for festival goers, without the need to search through social media. Event schedule, map, artists—it's all in one place! What's more, thanks to integration with the organizer's Facebook account, we ensure that you will always stay up-to-date!
```
gramatyka + stylistyka

Author: + You z dużej jako forma grzecznościowa, stosowałam się bardziej do ogólnych zasad w Polsce, wiem że w samym angielskim raczej się nie stosuje 
+ The App jest generalnie poprawne, pisze o tej aplikacji, ale jeśli this wygląda ładniej można zmienić, whatever
+ Bardziej mi pasuje All of that (of chyba mi uciekło w nocy) niż it's all in one place
+ używanie myślników zamiast przecinków jest znakomite😠
+ co do reszty - racja

Reviewer: > używanie myślników zamiast przecinków jest znakomite😠

ogólnie się zgadzam, tutaj mogłoby być i tak i tak

pozostałe kwestie mi (jako native speaker angielskiego) się wciąż rzucają jako nienaturalne no ale jak chcesz zostawić to zostaw. Wydaje mi się, że Polacy będą czytać wersję polską, więc wersja angielska nie powinna się sugerować zasadami j. polskiego

Author: Dobra kapitanie, to zrobimy tak - You zmienie na you, bo mnie przekonałeś.

> Bardziej mi pasuje All of that (of chyba mi uciekło w nocy) niż it's all in one place

Co do tego to tutaj bardziej chodzi o sens niż o gramatykę, jakoś tak medialnie bardziej do mnie trafia, jeśli masz jakiś pomysł jak to uładnić to spoko, ale it's all in one place jest dosyć basic.

Reviewer: chyba akurat zedytowałem komentarz po twoim, tam mam sugestię co do tego

Author: Jak szczegółowo😭 Jako native speaker polskiego nigdy w życiu bym się nie podjęła tłumaczenia ludziom jego gramatyki.

## Our team

- [@JanGolenski](https://github.com/JanGolenski) - Project Manager
- [@Octopus4344](https://github.com/Octopus4344) - Frontend Techlead
- [@Adriskk](https://github.com/Adriskk) - UI/UX Designer, Frontend Developer
- [@basia2137](https://github.com/basia2137) - UI/UX Designer
- [@kguzek](https://github.com/kguzek) - Frontend Developer
- [@jedryczkatymon](https://github.com/jedryczkatymon) - Frontend Developer
- [@GTR1701](https://github.com/GTR1701) - Frontend Developer
- [@zeolsem](https://github.com/zeolsem) - Frontend Developer
- [@Konzum59](https://github.com/Konzum59) - Frontend Developer

## Technologies used

- Next.js

> - React.js

Author: Czemu? Korzystamy z reacta, next to tylko framework + nie wszyscy go kojarzą

- TypeScript
- Tailwind CSS
- Coolify

## Links

[![docs.solvro.pl](https://i.imgur.com/fuV0gra.png)](https://docs.solvro.pl)

> # Development

Author: No to jest powiedzmy nowa sekcja już zupełnie ale no można i guess

### 1. Clone the Repository

```bash
git clone https://github.com/Solvro/web-juwenalia.git
```

### 2. Install Dependencies

```bash
cd web-juwenalia
npm install
```

### 3. Configure Environment

Create an `.env` file with the following content:

> ```env

Reviewer: wydaje mi się że nie ma takiego języka jak `env`, a pliki `.env` używają syntaktyki języka bash, bo tak właśnie definiuje się zmienne w systemach opartych na Unix, dlatego można m.in. użyć komendy `source .env`

> ```env

Author: Można ALE mam poważny argument za użeciem env, mianowicie edytory wlapuje to i zmieniaja kolorki na envowe co jest super giga

FACEBOOK_ACCESS_TOKEN=
FACEBOOK_APP_ID=
FACEBOOK_APP_SECRET=
```

### 4. Run the Project

```bash
npm run dev
```

### 5. View the Application

Open your browser and navigate to [http://localhost:3000](http://localhost:3000).

## Working with git

### Github Solvro Handbook

https://docs.solvro.pl/guides/github

### SSH

> If you are a Windows, follow this [tutorial](https://www.youtube.com/watch?v=vExsOTgIOGw) to connect via SSH

Author: O 3 w nocy sie skipuje wyrazy😭

### Building a new feature

1. Checkout and update main branch

```bash
   git checkout main
   git pull origin main
   git fetch
```

2. Create new feature branch

```bash
   git checkout -b WEB-x_my_feature_branch
```

> x - stands for issue number; it is going to checkout and create new branch name WEB-x_my_feature_branch

3. Commit your changes:

```bash
   git add .
   git commit -m "<description>"
```

4. Push to remote:

```bash
   git push origin WEB-x_my_feature_branch
```

5. Create a Pull Request on GitHub and wait for a review

### ⚠️ Important ⚠️

- Do not push directly to main branch!
- Please remember to commit before checking out to a different branch
- Clean up after a successful merge

  ```bash

>   git branch -d WEB-x_my_feature_branch

Author: Co do tego muszę się zorientować, bo takie opisy były stosowane do tej pory w readme solvro
